spec/requests/v1/admin_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe 'Admin' do
+  describe '#commit' do
+    it 'returns successful response' do
```

Comment:
@cgalarza could you please add a second test that shows a failing response when doing a post without auth?